### ISSUE TITLE
Windows Client Not Opening At All

### ISSUE THREAD

**Issue by fried-tantrum** [nextcloud/desktop#3385]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Steps to reproduce
1.
2.
3.

### Client configuration
Client version:

Operating system:

OS language:

Qt version used by client package (Linux only, see also Settings dialog):

Client package (From Nextcloud or distro) (Linux only):

Installation path of client:

### Expected behaviour
The client should start.

### Actual behaviour
Literally nothing.

### Steps to reproduce
1. Install the latest version of Nextcloud
2. Click the desktop icon.

### Client configuration
Client version: 3.2.2 (Also happened on 3.2.1)

Operating system: Windows 10, 21H!, Build 19043.985

OS language: English

Installation path of client: C:\Program Files\Nextcloud

I'm happy to provide any logs, screenshots, etc, but I'm not really an expert so you may have to tell me where to find the relevant logs.

**Comment by FlexW** [comment]
Can you find logs in C:\Users\YOURUSERNAME\AppData\Roaming\Nextcloud\logs and post them? Did it ever work in the past? If yes, when did it stop to work?

**Comment by fried-tantrum** [comment]
1) It appears I don't have a Nextcloud log. I went to C:\Users\MYUSERNAME\AppData\Roaming - no Nextcloud directory. Very odd. I downloaded the installer and ran it, it didn't pop up any issues for me, and the icon is on my desktop. Not sure why it didn't make a directory.

2) This is a new (refurbished) computer. On this particular machine, no it has never worked. However, before this I was running Windows 10 20H2 on a different machine (possibly 21H1, I just made the move a few days ago and I didn't bother checking where I left off on the old machine) and it was working just fine with no problems back then. It's likely that I installed a slightly older version of Nextcloud when I set that machine up, but I did keep it regularly updated, both Nextcloud and Windows. Unfortunately I'm unable to go check that machine's settings because I did a fresh USB install of Windows and handed it off to someone else.

**Comment by Byorun** [comment]
My client blocks the whole windows explorer after reboot (or frist start, doesn't matter). 
The tray icon stays grey and won't react to anthing. I have to kill the nextcloud client via task manager in order to be able to use my windows.

```
2021-06-05 14:37:07:557 [ info nextcloud.gui.application ]:	"################## Nextcloud locale:[de_DE] ui_lang:[] version:[3.2.2stable-Win64 (build 20210527)] os:[Windows 10 (10.0)]"
2021-06-05 14:37:07:562 [ info nextcloud.gui.application ]:	Using "en_GB" translation
2021-06-05 14:37:07:568 [ info nextcloud.gui.application ]:	VFS windows plugin is available
2021-06-05 14:37:07:568 [ info nextcloud.gui.application ]:	VFS suffix plugin is available
2021-06-05 14:37:07:568 [ debug nextcloud.gui.socketapi ]	[ OCC::SocketApi::SocketApi ]:	creating "//./pipe" true
2021-06-05 14:37:07:568 [ info nextcloud.gui.socketapi ]:	server started, listening at  "\\\\.\\pipe\\nextcloud-Byorun"
2021-06-05 14:37:07:568 [ info nextcloud.gui.folder.manager ]:	setting remote poll timer interval to 30000 msec
2021-06-05 14:37:07:583 [ info nextcloud.gui.account.manager ]:	Account for QUrl("https://********.net") using auth type "webflow"
2021-06-05 14:37:07:583 [ info nextcloud.sync.credentials.webflow ]:	Get QNAM
2021-06-05 14:37:07:583 [ info nextcloud.gui.account.manager ]:	Restored:  0  unknown certs.
2021-06-05 14:37:07:584 [ debug nextcloud.gui.activity ]	[ OCC::User::setNotificationRefreshInterval ]:	Starting Notification refresh timer with  300  sec interval
2021-06-05 14:37:07:605 [ info nextcloud.gui.folder.manager ]:	Setup folders from settings file
2021-06-05 14:37:07:605 [ info plugins ]:	Created VFS instance for: "win"
2021-06-05 14:37:07:605 [ info nextcloud.sync.database ]:	Detected filesystem "NTFS" for "D:/Nextcloud/.sync_b02f1d327368.db"
2021-06-05 14:37:07:605 [ debug nextcloud.gui.folder ]	[ OCC::Folder::checkLocalPath ]:	Checked local path ok
2021-06-05 14:37:07:606 [ info nextcloud.sync.configfile ]:	Adding user defined ignore list to csync: "C:/Users/Byorun/AppData/Roaming/Nextcloud/sync-exclude.lst"
```

the log stopps there. I already tried to reinstall the client with deleting all nextcloud folders under %AppData% but this does nothing.

**Comment by Hsauer** [comment]
I have the same issue here. The nextcloud freezes and locks the Windows Explorer with it.
I noticed that the .db-wal file on the root folder of the sync got huge (>1GB, and up to 4GB in one situation). The NC client will stop freezing if I delete those files, and thus loose all the sync history.
Once I deleted the files, the client worked normally only for a few hours.

OS: W10 Pro 20H2
Client version: 3.2.2

I downgraded the NC client to the 3.1 version and the problem disappeared.

**Comment by staggindraggin** [comment]
Can confirm downgrading to 3.1 solves this issue and the windows explorer not launching and crashing when opening the context menu issues I was having. I was able to stop the explorer crashing in 3.2 by renaming NCContextMenu.dll and NCOverlays,dll files.

**Comment by fried-tantrum** [comment]
> 
> 
> Can confirm downgrading to 3.1 solves this issue and the windows explorer not launching and crashing when opening the context menu issues I was having. I was able to stop the explorer crashing in 3.2 by renaming NCContextMenu.dll and NCOverlays,dll files.

Unfortunately this does not seem to have solved my issue. I uninstalled 3.2 (weird that it won't open but the system will find it as an installed program), installed 3.1.3 from here (https://download.nextcloud.com/desktop/releases/Windows/), same result. No Nextcloud folder in the directory listed above, nothing.

Please note, to everyone saying "I'm having this same problem," you're not describing the issue I'm experiencing. People keep talking about the client or Explorer freezing up. Nothing freezes up for me, nothing happens at all. It's as if I didn't even click on it. I'm tying this comment after my attempts to launch 3.1.3 just now, so clearly my machine isn't frozen up. I've tried running it as admin, repairing, reinstalling, uninstalling and reinstalling, I've tried just about everything I can think of. There is no client for me to log into, nothing happens. No login window pops up, no quicklaunch icon or folder to sync. Literally nothing happens. As I said, it's as if I didn't even click the icon. I've tried opening from Desktop, Start Menu, right click -> open file location and opening from there. Just nothing.

**Comment by ssaavedra** [comment]
This is happening to me on 3.2.2. I installed it both from either donwload.nextcloud.com or from chocolatey.

On the first run, I configure my host and enable VFS support. Everything is fine, except for the .sync*-db-wal file getting absurdly large.
After I quit Nextcloud and start it up again, it will not show any UI, but it will make the explorer shell wonky: right-click will freeze explorer, and opening new explorer windows freezes explorer.exe some times as well.

Latest log on $APPDATA/Roaming/Nextcloud/logs is very succint (attached).

[20210615_1518_owncloud.log.txt](https://github.com/nextcloud/desktop/files/6655883/20210615_1518_owncloud.log.txt)

If I downgrade to 3.1.3 (`choco install nextcloud-client --version=3.1.3 --allow-downgrade`) the UI will work but will show no synchronized folders (presumably because of the lack of VFS support).

**Comment by ssaavedra** [comment]
I have revoked the session from Nextcloud and the Windows client still freezes, so the freeze happens *before* the connection to the nextcloud server occurs. Is there a better way I can debug this?

Using `--logdebug` on launch does not seem to change what the log files output at all.

**Comment by Hsauer** [comment]
The freezing problem seems to be related to the size of the  .sync*-db-wal
file, possibly when the client tries to read the database (?). When I
downgraded the client, the version 3.1 would still freeze with those huge
db files.
Once the files were erased the older version worked normally.


Em ter., 15 de jun. de 2021 às 10:58, Santiago Saavedra <
***@***.***> escreveu:

**Comment by ssaavedra** [comment]
It might not be due to its size. In my case, after disabling VFS, I get three files:
- a .sync*.db
- a .sync*.db-shm, and 
- a .sync*.db-wal

However, when using VFS I only have .sync*.db and .sync*.db-wal, it might also be related to that. Also, my WAL file when not using VFS is smaller but not that much (around 23MB).

**Comment by fried-tantrum** [comment]
I have an update on this. So eventually I went as far back as I could using the link I shared above (https://download.nextcloud.com/desktop/releases/Windows/) and installed a MUCH older version (I think 2.2.3.4, but I could be wrong). That worked and I was able to sync again. However, the other day, it auto-updated to the latest and now I'm back where I was: it won't open at all, and weirdly I still have no Nextcloud folder under the AppData directory from the first comment. I notice that my Linux side updated 3.2.3 this morning, so whenever Windows has that release I'll update and see if that fixes it. In the meantime I guess I'll roll back to 2.2.3.4 and tell it not to update.
